src/lxc/conf.c

```diff
@@ -1569,7 +1569,7 @@ static int lxc_setup_dev_console(const struct lxc_rootfs *rootfs,
 	if (console->path && !strcmp(console->path, "none"))
 		return 0;
 
-	ret = snprintf(path, sizeof(path), "%s/dev/console", rootfs->mount);
+	ret = snprintf(path, sizeof(path), "%s/dev/console", rootfs->path ? rootfs->mount : "");
```

Comment:
Could you please perform this check atop the function once:

    char rootfs_path = rootfs->path ? rootfs->mount : "";

and then just use `rootfs_path` everywhere, please?

Reply:
Fixed